packages/jupyterlab-gridstack/src/editor/gridstack/model.ts

```diff
@@ -174,14 +191,20 @@ export class GridStackModel {
    */
   set info(info: DashboardView) {
     this._info = info;
-    const data = this._context.model.metadata.get('extensions') as Record<
+    /* const data = this._context.model.metadata.get('extensions') as Record<
```

Comment:
A minor cleanup follow-up could be to remove this comment and a couple of others.

Reply:
Ouch! Opening an issue!